zulipterminal/helper.py

```diff
@@ -629,3 +632,37 @@ def display_error_if_present(response: Dict[str, Any], controller: Any
                              ) -> None:
     if response['result'] == 'error' and hasattr(controller, 'view'):
         controller.view.set_footer_text(response['msg'], 3)
+
+
+def open_media(controller: Any, media_link: str, message_id: int) -> None:
+    """
+    Downloads media in /tmp/zulip-term/{message_id} directory and uses native
+    viewer for the output.
+    """
+    img_name = media_link.split('/')[-1]
+    target_dir = os.path.join(gettempdir(), 'zulip-term', str(message_id))
+    img_path = os.path.join(target_dir, img_name)
+
+    try:
+        os.makedirs(target_dir, exist_ok=True)
+    except FileExistsError:
+        # Raised when the target path exists and is not a directory.
+        # FIXME: This is highly unlikely. Do we need to handle this?
+        return
```

Comment:
While the zulip-term namespacing may be useful, I think this limits the possibility of multiple users saving files?

`NamedTemporaryFile` seems like a simpler approach, with appropriate prefixes and suffixes?